src/test/unit_tests/test_bucketspace.js

```diff
@@ -42,8 +42,14 @@ mocha.describe('bucket operations - namespace_fs', function() {
         region: 'us-east-1',
         httpOptions: { agent: new http.Agent({ keepAlive: false }) },
     };
-
-    mocha.before(async () => fs_utils.create_fresh_path(tmp_fs_root));
+    mocha.before(function() {
+        if (process.getgid() !== 0 || process.getuid() !== 0) {
+            coretest.log('No Root permissions found in env. Skipping test');
+            this.skip(); // eslint-disable-line no-invalid-this
+        }
+    });
+    mocha.before(async () => fs_utils.create_fresh_path(tmp_fs_root, 0o777));
+    mocha.before(async () => fs_utils.create_fresh_path(tmp_fs_root + bucket_path, 0o770));
```

Comment:
can you explain - why 770?

Reply:
I want that `other` users won't be able to access the directory.

Reply:
ok got it